scripts/linux/config-etc-hosts.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+
+SCRIPT_NAME="manage-hosts.sh"
+SCRIPT_PATH="/vagrant/scripts/linux"
```

Comment:
See above. Let me think if we can get rid of this. Also, this is a duplicate...

Reply:
ok, see my comment at the end